rplugin/python3/deoplete/__init__.py

```diff
@@ -1,4 +1,4 @@
-#=============================================================================
+# ============================================================================
```

Comment:
Yeah, coding style: it's a separate commit, and I've noticed then that there are too many, and stopped before installing `autopep8`.  I've not noticed that there are so many Python files in the beginning.